src/css/variablesCustom.css

```diff
@@ -180,82 +180,13 @@
   --footer-default-footer-margin: 0 auto;
   --footer-default-footer-width-mobile: 100%;
   --footer-default-footer-margin-mobile: 0;
-  --p-margin: 0 0 calc(16rem/18) 0; /* 16px */
```

Comment:
please check it at page home to see the sideeffect. To make sure, that stuff within body > main is aligned in center, they must have margin-left/right auto. Otherwise the body alignment with width and margin auto does not work. This is true for every element which can be directly placed within body > main.

in this case and the one below: ul and p nodes